presto-expressions/pom.xml

```diff
@@ -31,6 +31,11 @@
             <artifactId>presto-common</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-parser</artifactId>
```

Comment:
Execution code in workers should not depends on the parser. Let's move the helper function to a different location to avoid introducing this dependence.

@highker for suggestions on better places to put the helper functions in LogicalRowExpressions.java.

Reply:
`com.facebook.presto.sql.analyzer.ExpressionTreeUtils` seems somehow related